README.md

```diff
@@ -38,7 +38,7 @@ use skywalking::context::trace_context::TracingContext;
 use skywalking::reporter::grpc::Reporter;
 use tokio;
 
-async fn handle_request(reporter: ContextReporter) {
+async fn handle_request(reporter: Reporter) {
```

Comment:
I think we should add a new API's doc here.

Reply:
Maybe we can integrate `Tracer` and `Reporter`, like in go.

Reply:
Yes, agree. 
And whether they should be const, most likely user's choice, right?
We don't have to require that.